test/scripts/find_subnets.py

```diff
@@ -0,0 +1,47 @@
+import ipaddress
+import argparse
+import math
+import sys
+
+def find_subnets(network, required_subnets):
+    try:
+        # Create additional subnets to avoid overlapping
+        required_subnets += 2
```

Comment:
We never needed to leave additional pool of addresses to avoid overlapping, and we have this weird logic only because I copied and reworked scripts from Istio, because I wanted to achieve similar multi-network setup. The original script was taking a subnet and allocated 100 (?) last addresses "to avoid overlapping". Integrating Istio test framework was complicated enough to forget about investigating why the original author claim that addresses may overlap. Now, we can remove it, because the default KinD installation does not create any LB service, and even if the cluster was provisioned with a LB service, it wouldn't have assigned any IP until MetalLB is installed, so there is no risk of overlapping IP addresses.
```suggestion
        # Create additional subnets to avoid overlapping
        required_subnets += 2
```

Reply:
> Now, we can remove it

I am a bit puzzled by your suggestion then :)

Reply:
Anyway, thanks for the background story. Fixed in 89affcd